deploy/template.yaml

```diff
@@ -1095,6 +1095,9 @@ Resources:
                 {
                     "message": "Not Found"
                 }
+        RequestParameters:
```

Comment:
Should this not go in IntegrationResponses[0].ResponseParameters see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-apigateway-method-integrationresponse.html#cfn-apigateway-method-integrationresponse-responseparameters. As RequestParameters looks like it goes to S3 https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-apigateway-method-integration.html#cfn-apigateway-method-integration-requestparameters.

Reply:
Right you are, these have been moved.